README.md

```diff
@@ -9,11 +9,10 @@ deployment scheme.
 
 The components in this project are:
 
-* **kubernetes** (https://github.com/kubernetes/kubernetes/), the container management system.
-* **Glusterfs** (https://www.gluster.org/), the scale-out storage system, running as pods inside the kubernetes cluster.
+* **Kubernetes** (https://github.com/kubernetes/kubernetes/), the container management system.
+* **GlusterFS** (https://www.gluster.org/), the scale-out storage system, running as pods inside the kubernetes cluster.
 * **heketi** (https://github.com/heketi/heketi), Gluster's volume management service interface, running in a pod inside kubernetes.
 
-
 Resources:
 
-* [Documentation](./docs/index.md)
+* [Setup Guide](./docs/setup-guide.md)
```

Comment:
this hunk i do not agree with (without further arguments): I thought of having a slim little front page and having the people click further into documentation to select the doc they want to see, there. makes any sense at all?